### ISSUE TITLE
Úkoly k řetězcům: není ze začátku jasné co má být v rozsypanycaj.txt

### ISSUE THREAD

**Issue by encukou** [pyvec/naucse.python.cz#114]
Je to napsané až dole, mělo by to být hned na začátku.

**Comment by hroncok** [comment]
Mluvíš o [handout7.tex](https://github.com/PyLadiesCZ/pyladies.cz/blob/master/original/v1/s005-modules/handout/handout7.tex)? Neměla by ta issue být v [pyladies.cz](https://github.com/PyLadiesCZ/pyladies.cz) repu? Začínáme v tom mít docela bordel, co handouty přesunout sem?

**Comment by encukou** [comment]
Jo, je v tom trochu zmatek, ale radši bych řešil všechno k materiálům na jednom místě.
Přesunout by to bylo fajn, ale nejlíp převedené na Markdown :)
